test/Grpc.Net.Client.Tests/AsyncClientStreamingCallTests.cs

```diff
@@ -396,7 +396,7 @@ public override void Write(byte[] buffer, int offset, int count)
             throw new NotImplementedException();
         }
 
-#if !NET472_OR_GREATER
+#if !NET462_OR_GREATER
```

Comment:
What changed here?

Reply:
Update to tests to consistently test NET462 for .NET Framework.